vimeo-networking/src/test/java/com/vimeo/networking/model/appconfiguration/AppConfigurationTest.java

```diff
@@ -0,0 +1,18 @@
+package com.vimeo.networking.model.appconfiguration;
+
+import com.vimeo.networking.Utils;
+
+import org.junit.Test;
+
+/**
+ * Unit tests for {@link AppConfiguration}.
+ * <p>
+ * Created by restainoa on 4/20/17.
+ */
+public class AppConfigurationTest {
+
+    @Test
+    public void verifyTypeAdapterWasNotGenerated() throws Exception {
+        Utils.verifyNoTypeAdapterGeneration(AppConfiguration.class);
```

Comment:
Why doesn't `AppConfiguration` use Stag? It has `SerializedName` annotated members...

This question applies to other classes in this package.